test/Main.hs

```diff
@@ -0,0 +1,33 @@
+{-# LANGUAGE QuasiQuotes #-}
+
```

Comment:
we need `{-# LANGUAGE OverloadedStrings #-}` here to get the literals to compile